trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'Decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super.correct_name[0, 10]
+  end
+end
```

Comment:
👍 Your code is coming along nicely! I noticed that you're currently using the `super` keyword in your `TrimmerDecorator` class to call the parent class's `correct_name` method and then apply it trimming. While this approach works, there might be a more straightforward and intuitive way to achieve the same result.

- [ ] Using `super` assumes that the parent class (`Decorator`) will always implement the `correct_name` method in the expected way. This can lead to potential issues if the parent class changes or if you want to customize the behavior further. On the other hand, the below code example offers a more direct and explicit way of capitalizing the name. It accesses the `@nameable` object's `correct_name` method and then applies triming Here's what it looks like:

```ruby
class TrimmerDecorator< Decorator
  def correct_name
    @nameable.correct_name......
  end
end
```

By adopting this approach, you not only simplify your code but also make it more flexible and easier to maintain. Keep up the great work! 👏🚀